presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -245,6 +247,11 @@ protected String getTableSchemaName(ResultSet resultSet)
                     int scale = min(decimalDigits, getDecimalDefaultScale(session));
                     return Optional.of(decimalColumnMapping(createDecimalType(Decimals.MAX_PRECISION, scale), getDecimalRoundingMode(session)));
                 }
+                break;
+            case Types.BINARY:
+            case Types.VARBINARY:
+            case Types.LONGVARBINARY:
+                return Optional.of(ColumnMapping.sliceMapping(VARBINARY, (resultSet, columnIndex) -> wrappedBuffer(resultSet.getBytes(columnIndex)), varbinaryWriteFunction(), FULL_PUSHDOWN));
```

Comment:
Pleease extract `(resultSet, columnIndex) -> wrappedBuffer(resultSet.getBytes(columnIndex))` as a function in `StandardColumnMappings`, place it right below `varbinaryColumnMapping` there, and reuse in `varbinaryColumnMapping`.